.clang-format

```diff
@@ -0,0 +1,80 @@
+﻿---
+BasedOnStyle: Google
+AccessModifierOffset: -1
+AllowShortFunctionsOnASingleLine: Empty
+AllowShortBlocksOnASingleLine:    Empty
+BraceWrapping:
+  AfterCaseLabel:  false
+  AfterClass:      true
+  AfterControlStatement: true
+  AfterEnum:       true
+  AfterFunction:   true
+  AfterNamespace:  true
+  AfterObjCDeclaration: true
+  AfterStruct:     true
+  AfterUnion:      true
+  AfterExternBlock: true
+  BeforeCatch:     true
+  BeforeElse:      true
+  BeforeLambdaBody: true
+  BeforeWhile:     true
+  IndentBraces:    false
+  SplitEmptyFunction: false
+  SplitEmptyRecord: false
+  SplitEmptyNamespace: false
+BreakBeforeBraces: Custom
+ColumnLimit: 100
+PointerAlignment: Right
+SpacesInParentheses: true
+SpacesInSquareBrackets: true
+TabWidth: 4
+UseTab: AlignWithSpaces
+CommentPragmas:  '^ keep:'
+IncludeCategories:
```

Comment:
It might be better to avoid include sorting for now; I bet this causes a lot of compile errors because we have non-self-contained headers.